torchinductor/lowering.py

```diff
@@ -1321,10 +1321,18 @@ def full_like(x, fill_value, **kwargs):
 def tensor_constructor(fill_value):
     # torch.zeros, torch.ones, etc
     def inner(
-        *size, dtype=None, device=None, layout=0, pin_memory=False, memory_format=None
+        *size,
+        names=None,
+        dtype=None,
+        device=None,
+        layout=0,
+        pin_memory=False,
+        memory_format=None,
     ):
+        assert names is None
```

Comment:
Why does names have to be none? It just an unused arg, right?

Reply:
Yes, the FX graph aot generated has this attribute, but we don't use it.

Reply:
If someone calls a tensor ctor with `names` - `torch.zeros(2, 3, names=('N', 'C'))` - we won't use the names, but we will assert. However, this assert does not really protect correctness. Do we need it?

Reply:
It doesn't protect correctness, but it forces inductor to error out on things it can't handle.